src/c++/perf_analyzer/genai-pa/genai_pa/llm_inputs/llm_inputs.py

```diff
@@ -653,7 +653,7 @@ def _add_optional_tags_to_openai_json(
         if add_model_name:
             pa_json["data"][index]["payload"][0]["model"] = model_name
         if add_stream:
-            pa_json["data"][index]["payload"][0]["stream"] = [True]
+            pa_json["data"][index]["payload"][0]["stream"] = True
```

Comment:
openAI expects bool, not array.  The vllm/trtllm cases do expect array and should stay the way they were